package.json

```diff
@@ -29,20 +29,24 @@
     "@angular/platform-browser": "^5.0.0",
     "@angular/platform-browser-dynamic": "^5.0.0",
     "@angular/router": "^5.0.0",
-    "@deck.gl/core": "^7.1.6",
-    "@deck.gl/google-maps": "^7.1.6",
-    "@deck.gl/layers": "^7.1.6",
+    "@deck.gl/core": "^7.3.15",
+    "@deck.gl/google-maps": "^7.3.15",
+    "@deck.gl/layers": "^7.3.15",
     "@turf/bbox": "^6.0.1",
+    "@types/crypto-js": "^3.1.44",
     "@types/d3-scale": "^1.0.11",
     "@types/gapi": "0.0.35",
-    "@types/googlemaps": "^3.29.2",
+    "@types/googlemaps": "^3.38.0",
     "angular-webstorage-service": "^1.0.2",
-    "codemirror": "^5.34.0",
+    "codemirror": "^5.52.2",
     "colorbrewer": "^1.1.0",
-    "core-js": "^2.4.1",
-    "d3-color": "^1.0.3",
+    "core-js": "^2.6.11",
+    "crypto-js": "^4.0.0",
+    "d3-color": "^1.4.0",
     "d3-scale": "^1.0.7",
-    "js-file-download": "^0.4.1",
+    "firebase": "^7.13.1",
+    "firebase-admin": "^8.10.0",
```

Comment:
I don't see this `firebase-admin` dependency used anywhere, and it appears to be meant for "access to Firebase services from privileged environments (such as servers or cloud) in Node.js". I think we can remove it?

Reply:
Done.